### ISSUE TITLE
`flatpak unmask` as an alias for `flatpak mask --remove`

### ISSUE THREAD

**Issue by YaLTeR** [flatpak/flatpak#4019]
## Linux distribution and version
Fedora 33

## Flatpak version
Flatpak 1.8.3

## Description of the problem
Would be a nice alias I think.

**Comment by nekohayo** [comment]
Yes, it is particularly confusing since https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-mask and https://docs.flatpak.org/en/latest/tips-and-tricks.html do not mention any way to unmask. I had to find this issue to even know it was possible.
